inst/dummy_package/0105-run-birth-death.R

```diff
@@ -1,37 +0,0 @@
-#' # Simple birth-death difference equation model
```

Comment:
Why was this deleted? Don't the students need it for the second practical series?

Reply:
Oh, sorry - this is in dummy_package - it should be deleted.